src/libDirectoryService/Coinbase.cpp

```diff
@@ -137,8 +202,17 @@ void DirectoryService::InitCoinbase() {
   }
 
   LOG_MARKER();
+
+  VectorOfLookupNode vecLookup = m_mediator.m_lookup->GetLookupNodes();
```

Comment:
Hmm perhaps we can change `GetLookupNodes()` to return `const VectorOfLookupNode&`, then here (and everywhere else it is called) we just get the reference instead of making a new vector?

Reply:
There is a mutex involved with the var returned. so I am not sure

Reply:
I see, then better not change for now